Add debian to instructions

andyliuhaowen opened the pull request:
Although Debian is not as popular as a gaming setup, I think its general popularity and the upstream (wine)'s efforts in supporting it merit a mention at the end of the chart.

tannisroot commented:
Hi, sorry for a VERY late review, but better late than never

tannisroot commented on WineDependencies.md:
```diff
@@ -21,6 +21,9 @@ Add the repository:
 
 |For this version: | Use this command:          
 |------------------|--------------------------------
+|Debian 9 Jessie   | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ jessie main'
```
Debian 9 is too old for our Lutris wine builds

tannisroot commented on WineDependencies.md:
```diff
@@ -21,15 +21,15 @@ Add the repository:
 |Ubuntu 18.10      | sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ cosmic main'
 |Ubuntu 18.04<br>Linux Mint 19.x | sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ bionic main'
+|Debian 9 Jessie   | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ jessie main'
+|Debian 10 Buster  | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ buster main'
```
Correct me if I'm wrong, but I don't think add-apt-repository exists on Debian out of the box.

andyliuhaowen commented:
Thanks for the review!

I have removed Jessie from the list, and added the command to install `software-properties-common` for Buster.

Bullseye installs `software-properties-common` by default using the graphical installer, which I assume is applicable to almost all gamers. I also assume those who installed with other methods know enough to figure out how to get `apt-add-repository`.
